lib/index.ts

```diff
@@ -19,11 +19,15 @@ import { downloadUrl, hash, plusx } from './utils';
 import patchesJson from '../patches/patches.json';
 import { version } from '../package.json';
 
+const { PKG_FETCH_MIRROR } = process.env;
+
 async function download(
   { tag, name }: Remote,
   local: string
 ): Promise<boolean> {
-  const url = `https://github.com/vercel/pkg-fetch/releases/download/${tag}/${name}`;
+  const url = typeof PKG_FETCH_MIRROR !== "undefined" ?
```

Comment:
Perhaps even just a logical `OR` since if the variable is defined, we want to use it, and if it's undefined, it can fall back to the default, like:
`const baseUrl = PKG_FETCH_MIRROR || 'https://github.com/vercel/pkg-fetch/releases/download';`

In this case, you are pointing out kind of a base url to use, so perhaps it's a bit more readable, and reduces duplicated code to have something like:

```typescript
const baseUrl = PKG_FETCH_MIRROR || 'https://github.com/vercel/pkg-fetch/releases/download';
const url = `${baseUrl}/${tag}/${name}`;
```

Reply:
I'm also quite open to the idea of omitting the `${tag}` prefix for the mirror, since presumably you want to just simply point out where the binaries are directly, without relying on specific implementation assumptions in the code, like a tag subfolder in the download path.

Reply:
@wuliang142857 can you continue?